pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "ipumspy"
-version = "0.1.0"
+version = "0.1.1-alpha"
```

Comment:
We're hoping to put out version 0.2.0 in a few weeks. We can roll these changes in with that release.

Reply:
Thank you for all these suggestions and information! I'll work on them in the following days.